### PR TITLE
feat: apply feedback

### PR DESCRIPTION
**Pull request by skdud4659**
### To Reviewers
- [ ] Skip (`style`, `chore`, `ci`, minor refactoring, etc.)
- [ ] Need discussion
- [ ] Not that difficult
- [ ] Approved feature branch merge to master

### Description
SSIA

### Things to Talk About
Plz check by commits!

### CODE REVIEW

File: apps/web/src/lib/helper/config-data-helper.ts [hunk 1]
```diff
@@ -183,40 +187,89 @@ export const convertCostAnalysisConfigToReferenceData = (config: ConfigData[]|nu
...
+
+    Object.values(groupedResults).forEach((group) => {
+        results.push(...group);
+    });
+
     return results;
 };
 
-export const convertDashboardConfigToReferenceData = (config: ConfigData[]|null, dashboardList: DashboardModel[]): ReferenceData[] => {
+export const convertDashboardConfigToReferenceData = (config: ConfigData[]|null, dashboardList): ReferenceData[] => {
```
Comment (piggggggggy) [inline apps/web/src/lib/helper/config-data-helper.ts:243]: Question: Why did you remove type?

Comment (piggggggggy) [reply apps/web/src/lib/helper/config-data-helper.ts:243]: How about using `(PublicDashboardModel|PrivateDashboardModel)[]`?

File: apps/web/src/services/project/components/ProjectMainProjectGroupDeleteCheckModal.vue [hunk 2]
```diff
@@ -23,26 +23,30 @@ const projectPageGetters = projectPageStore.getters;
...
+            });
+        }
         await projectPageStore.deleteProjectGroup();
         await projectGroupStore.load({ force: true });
         showSuccessMessage(i18n.t('PROJECT.LANDING.ALT_S_DELETE_PROJECT_GROUP'), '');
-        await favoriteStore.deleteFavorite({
-            itemType: FAVORITE_TYPE.PROJECT_GROUP,
-            workspaceId: state.currentWorkspaceId || '',
-            itemId: state.groupId,
-        });
```
Comment (piggggggggy) [inline apps/web/src/services/project/components/ProjectMainProjectGroupDeleteCheckModal.vue:45]: Suggestion: I think existing position looks better~

**Review by piggggggggy — APPROVED** [review]
LGTM~ I left some comments

File: apps/web/src/services/project/pages/ProjectDetailPage.vue [hunk 3]
```diff
@@ -162,13 +163,16 @@ const projectDeleteFormConfirm = async () => {
...
         showSuccessMessage(i18n.t('PROJECT.DETAIL.ALT_S_DELETE_PROJECT'), '');
         router.go(-1);
+        const isFavoriteItem = favoriteGetters.projectItems.find((item) => item.itemId === projectDetailPageState.projectId);
+        if (isFavoriteItem) {
+            await favoriteStore.deleteFavorite({
+                itemType: FAVORITE_TYPE.PROJECT,
+                workspaceId: storeState.currentWorkspaceId || '',
+                itemId: projectDetailPageState.projectId as string,
+            });
+        }
```
Comment (yuda110) [inline apps/web/src/services/project/pages/ProjectDetailPage.vue:175]: praise: good!!!

**Review by yuda110 — APPROVED** [review]
good good
